### ISSUE TITLE
In folders, contact pictures function as check boxes or toggle buttons but are not presented to assistive technology as such 

### ISSUE THREAD

**Issue by dankeck** [thunderbird/thunderbird-android#8446]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App

Thunderbird for Android

### App version

8.0b5

### Where did you get the app from?

Google Play

### Android version

14

### Device model

Pixel 5

### Steps to reproduce

1. Open Thunderbird.
2. Ensure that "Show contact pictures" is enabled in Settings -> General Settings -> Display
3. Go to a mail folder.
4. Turn on the TalkBack screen reader.
5. Swipe to one of the contact pictures next to an email message.

### Expected behavior

- Contact picture is announced by TalkBack as a [check box](https://www.w3.org/WAI/ARIA/apg/patterns/checkbox) or [toggle button](https://www.w3.org/WAI/ARIA/apg/patterns/button), because it can be toggled and added to a selection, in order to perform actions on multiple messages, such as move, copy, add star, or mark as spam. 
- TalkBack announces the checked or pressed state of the control.

### Actual behavior

- Contact picture is announced by TalkBack as "Unlabeled. Double tap to activate," so the user does not know what tapping will do or even what it looks like.
- TalkBack does not announce whether the current email is part of the selection or not.

**Comment by kewisch** [comment]
Thanks for reporting! This seems like an a11y issue in the critical path, let's see if we can get it fixed.

**Comment by joohnq** [comment]
https://github.com/user-attachments/assets/2ccf58bc-769f-45aa-9fd2-8a248c63c042

Is this the behavior that you want?
